metadata_tools/hosts/GO_0xxx/gcp_geometry_startup.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+apt-get update -y
+apt-get install -y python3 python3-pip python3-venv git
+cd
+git clone https://github.com/SETI/rms-metadata-tools.git
+cd rms-metadata-tools || exit 1
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+python3 GO_0xxx_geometry_cloud.py $RMS_METADATA/GO_0xxx/ $RMS_METADATA/GO_0xxx/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)